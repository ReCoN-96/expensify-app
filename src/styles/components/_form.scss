.form {
  display: flex;
  flex-direction: column;
  > * {
    margin-bottom: $m-size;
    > div > .SingleDatePickerInput {
      width: 100%;
    }
  }
}

.form__error {
 margin: 0 0 $m-size 0;
 font-style: italic;
}