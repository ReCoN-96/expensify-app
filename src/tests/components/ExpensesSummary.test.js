import React from 'react';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import { shallow } from 'enzyme';

test('Should correctly render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary totalAmountExpenses={1} sumOfExpenses={85} />)
  expect(wrapper).toMatchSnapshot();
});

test('Should correctly render ExpenssesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary totalAmountExpenses={25} sumOfExpenses={855} />)
  expect(wrapper).toMatchSnapshot();
});