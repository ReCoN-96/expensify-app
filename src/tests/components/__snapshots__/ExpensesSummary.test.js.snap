// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;

exports[`Should correctly render ExpenssesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;
